src/components/singleRecord.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import Row from 'react-bootstrap/Row'
+import Col from 'react-bootstrap/Col'
+import DropdownMenu from './dropdownMenu'
+function SingleRecord() {
+  return (
+    <>
+      <Row style={{ marginTop: '20px' }}>
+        <Col> Ahmed </Col>
+        <Col> 96 </Col>
+        <Col> Math </Col>
+        <Col>
+          <div className="gradeBox" style={{ color: '#ffb59d' }}>
```

Comment:
try to avoid inline styling if you're also use className too.

Reply:
did this for UI right now only, will make it dynamic later on via props